spring-cloud-schema-registry-core/src/main/java/org/springframework/cloud/schema/registry/model/Schema.java

```diff
@@ -46,8 +53,12 @@
 	@Column(name = "FORMAT", nullable = false)
 	private String format;
 
-	@Lob
-	@Column(name = "DEFINITION", nullable = false, length = 8192)
+	@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "subject")
+	@JsonIdentityReference(alwaysAsId = true)
+	@ManyToMany
+	private List<Schema> references = new ArrayList<>();
+
+	@Column(name = "DEFINITION", columnDefinition = "text", nullable = false, length = 8192)
```

Comment:
Removed `@Lob` annotation as it was creating problems during fetching `references` field during schema validation by throwing:
```
org.postgresql.util.PSQLException: Large Objects may not be used in auto-commit mode.
```
while using PostgreSQL.

It was replaced by `columnDefinition = "test"` and effectively creates the same type of column for `definition` field.

References:
https://shred.zone/cilla/page/299/string-lobs-on-postgresql-with-hibernate-36.html
```